### PR TITLE
[17.0][MIG] sale_order_line_position: Migration to 17.0 

### PR DESCRIPTION
**Pull request by xaviedoanhduy**

### CODE REVIEW

**Review by gurneyalex — APPROVED** [review]

File: sale_order_line_position/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        self.recompute_positions()
+        return super().action_confirm()
+
+    def action_quotation_send(self):
+        self.recompute_positions()
+        return super().action_quotation_send()
+
+    def recompute_positions(self):
+        for sale in self:
+            if sale.locked_positions or sale.company_id.disable_sale_position_recompute:
```
Comment (angelmoya) [inline sale_order_line_position/models/sale_order.py:27]: maybe could be better with filtered in for

File: sale_order_line_position/models/sale_order_line.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            WHERE order_id in %s GROUP BY order_id;
+            """
+            self.env.cr.execute(query, (ids,))
+            default_pos = {key: 1 for key in ids}
+            existing_pos = {
+                order_id: pos + 1 for order_id, pos in self.env.cr.fetchall()
+            }
+            sale_pos = {**default_pos, **existing_pos}
+            for line in vals_list:
+                if not line.get("display_type"):
```
Comment (angelmoya) [inline sale_order_line_position/models/sale_order_line.py:50]: same comment, better with filtered

**Review by angelmoya — APPROVED** [review]
Functional review LGTM

**Review by mmequignon — APPROVED** [review]

**Pull request merged**